### ISSUE TITLE
Validation & Nomination screen

### ISSUE THREAD

**Issue by bedeho** [Joystream/pioneer#3819]
We currently have nothing here, but there is a very serious need to make this process better. We always imagined Polkadot apps could be used, but that product not only evolves to focus on various changes that have occurred that are not relevant to us (e.g. nomination pools), but more importantly solves the identity issue using the `identity`pallet, which Joystream does not have. This makes it difficult for nominators to figure out what real world actor is associated with a given validator.

**Comment by traumschule** [comment]
@dmtrjsg i can take care of the data part. Is this a suitable task for design applicants?

**Comment by dmtrjsg** [comment]
I'd say only for successful ones, after they get used to the flow with working on these issues first:

<img width="665" alt="Screenshot 2022-11-21 at 18 04 57" src="https://user-images.githubusercontent.com/93924525/203074698-51aa43f6-80e3-4127-aedf-19d273c0d4ae.png">

**Comment by bedeho** [comment]
Discussion here got sidetracked, replacing with fresh issue https://github.com/Joystream/pioneer/issues/4105.

**Comment by traumschule** [comment]
@chrlschwb @ninja-fire Let's keep collecting design ideas here.

> can you tell me the most necessary data that should be displayed on the validator & nominator page of Pioneer?

The first step is to replicate the "live" staking view from [p-js](https://polkadot.js.org/apps/?rpc=wss%3A%2F%2Frpc.joystream.org#/staking) with
- key or if available member handle / team
- total stake
- own stake
- nominated stake
- commission
- reward points this era
- last block
- return
- button to see stats

This basically joins "Overview" and "Targets". Because it's a lot of information the design needs to find a smart way to not only show a simple table with rows but multi-line cards.
Identifying info like key / name and performance, return in the first while staking related info can go into the second line.
Clicking on a card / row item show mark the validator for nomination and activate a button "Nominate".
A down arrow to the right or left can be used to expand further info. For example to see further stats or graphs.

Teams data like labels and keys can be taken [from notion](https://joystream.notion.site/Meet-Your-Validators-a4a1d8dd629d4c9fa49cdfb3e252b86d) and stored as json hosted somewhere, for example a channel (#54). However for testing i suggest to go with a static version read from a forum post.

**Comment by ninja-fire** [comment]
do we already have something designed for the validators page?
I see there is a "validators" page coming soon.
<img width="221" alt="image" src="https://user-images.githubusercontent.com/44232936/214709162-472e9992-eec2-440c-88e4-4d80826fb247.png">

so what is the current state of this page?
If there is nothing designed yet, we need to define all the actions that can be done there and which information we want to display.

for the different actions, we need to define clear actions from validators and actions from nominator. 

Validators should also be able to fill in personal information like in this notion board:
<img width="1503" alt="image" src="https://user-images.githubusercontent.com/44232936/214709829-414e4284-63c8-4366-ba5f-670d69d7a035.png">

and nominators should be able to read validators' information and select one validator from a list. 

Please tell me if I'm wrong or missing information.

**Comment by bedeho** [comment]
> do we already have something designed for the validators page?
> so what is the current state of this page?

No, nothing, this disabled button was just left as a hint.

> If there is nothing designed yet, we need to define all the actions that can be done there and which information we want to display.

I would encourage you to not focus on actions or info, but user goals. The appropriate target audience user needs to be consulted to understand what is most important.

**Comment by bedeho** [comment]
I don't know why this issue was reopened, when it was superceeded by another issue, but there is a reply from an actual validator on the other issue

https://github.com/Joystream/pioneer/issues/4105#issuecomment-1403922162

I suggest we close this again.

**Comment by traumschule** [comment]
@bedeho Pleas wait. I worked quite a while on this last year already and will submit a PR when i get to it. Note that the scope of both isn't congruent, also design can happen in parallel.
